runtime/runtime/src/lib.rs

```diff
@@ -252,12 +252,12 @@ impl Runtime {
             }
         }
 
-        if !check_rent(&signer_id, &mut signer, &self.config, apply_state.epoch_length) {
+        if !check_rent(&signer_id, &signer, &self.config, apply_state.epoch_length) {
             return Err(format!("Failed to execute, because the account {} wouldn't have enough to pay required rent", signer_id).into());
         }
 
         match access_key.permission {
-            AccessKeyPermission::FullAccess => Ok(()),
+            AccessKeyPermission::FullAccess => (),
```

Comment:
This looks suboptimal. Why not use `if let` here?

Reply:
good point. It happened after a bunch of refactoring.